projects/ontimize-web-ngx/src/lib/layouts/form-layout/tabgroup/o-form-layout-tabgroup.component.ts

```diff
@@ -429,4 +429,13 @@ export class OFormLayoutTabGroupComponent implements OFormLayoutManagerMode, Aft
     return Object.keys(tabData.innerFormsInfo).some(formAttr => tabData.innerFormsInfo[formAttr].modified);
   }
 
+  closeDetails(detailsKeysData: any[], options?: FormLayoutCloseDetailOptions) {
+    detailsKeysData.forEach((detailData) => {
```

Comment:
protected this code to avoid undefined values or just initialize method argument as an empty array in case of not defined